utils/generate_tables.py

```diff
@@ -33,20 +33,43 @@ def get_data_size_and_hpss(hpss_path: str) -> Tuple[str, str]:
             hpss = ""
         return (data_size, hpss)
 
-def get_esgf(source_id: str, model_version: str, experiment: str, ensemble_num: str, cmip_only: str) -> str:
+def get_esgf(source_id: str, model_version: str, experiment: str, ensemble_num: str, link_type: str) -> str:
     if experiment and ensemble_num:
         # See https://github.com/E3SM-Project/CMIP6-Metadata/pull/9#issuecomment-1246086256 for the table of ensemble numbers
-        # remove v from model_version
-        esgf = f"`Native <https://esgf-node.llnl.gov/search/e3sm/?model_version={model_version[1:]}_0&experiment={experiment}&ensemble_member=ens{ensemble_num}>`_"
+        # Note that `[1:]`` removes `v` from `model_version`
+        # 
+        # Query parameters:
+        # v2:
+        # Native (model_version, experiment, ensemble_member)
+        # CMIP6 (source_id, experiment_id, variant_label)
+        # v1 on https://e3sm.org/data/get-e3sm-data/released-e3sm-data/v1-1-deg-data-cmip6/:
+        # Native (experiment, and optionally model_version)
+        # CMIP6 (activity_id, source_id, experiment_id)
+
+        # TODO: figure out if we're returning the correct things
+        # Comparison to links on https://portal.nersc.gov/project/e3sm/forsyth/data_docs/html/v1/WaterCycle/simulation_data/simulation_table.html
+        # simulation | e3sm.org CMIP6 | e3sm.org Native | data_docs CMIP6 | data_docs Native |
+        # piControl              | 346   | 67 | 346  | 15 |
+        # historical             | 3414 | 190 | 1420 | 70 | 5 rows on data_docs
+        # 1pctCO2                | 339  | 38  | 339  | 14 |
+        # abrupt-4xCO2           | 293  | 50  | 0    | 14 | 2 rows on data_docs
+        # abrupt-4xCO2-ext300yrs | x    | 14  |
+        # amip                   | 527  | 30  | 527  | 30 | 3 rows on data_docs
+        # amip_1850_allF         | x    | 30  |
+        # amip_1850_aeroF        | x    | 30  |
+        # TODO: only return Native for some experiments...
```

Comment:
https://e3sm.org/data/get-e3sm-data/released-e3sm-data/v1-1-deg-data-cmip6/ shows CMIP6 and Native for piControl, historical, 1pctCO2, abrupt-4xCO2, amip, but only Native for abrupt-4xCO2-ext300yrs, amip_1850_allF, amip_1850_aeroF.